enclave_apps/oak_functions_enclave_app/src/main.rs

```diff
@@ -47,28 +46,15 @@ fn main() -> ! {
     }
     let mut invocation_stats = StaticSampleStore::<1000>::new().unwrap();
 
-    let restricted_kernel_dice_data = {
-        let mut result = oak_dice::evidence::RestrictedKernelDiceData::new_zeroed();
-        let buffer = result.as_bytes_mut();
-        let len = oak_restricted_kernel_interface::syscall::read(
-            oak_restricted_kernel_interface::DICE_DATA_FD,
-            buffer,
-        )
-        .expect("couldn't read DICE data");
-        if len != buffer.len() {
-            panic!("invalid dice data size");
-        }
-        result
-    };
-    let encryption_key =
-        oak_crypto::encryptor::EncryptionKeyProvider::try_from(&restricted_kernel_dice_data)
-            .expect("couldn't get encryption key");
-    let evidence = oak_remote_attestation::proto::oak::attestation::v1::Evidence::try_from(
-        restricted_kernel_dice_data.evidence,
-    )
-    .expect("couldn't get evidence");
-    let service =
-        oak_functions_service::OakFunctionsService::new(evidence, Arc::new(encryption_key), None);
+    let encryption_key_handlee = oak_restricted_kernel_sdk::InstanceEncryptionKeyHandle::create()
+        .expect("couldn't encryption key");
+    let evidencer =
```

Comment:
Evidencer? :sweat_smile: I missed that in the SDK PR. Perhaps `evidenceProvider` would read better?

Reply:
Fair. I'll patch the SDK in a separate renaming PR.

Reply:
+1 to `EvidenceProvider`

But we also need to update the SDK document, if we do any changes to naming/call conventions